x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -157,6 +189,57 @@ export default function ({ getService }: FtrProviderContext) {
         await ml.securityUI.logout();
       });
 
+      describe('Add trained model flyout', () => {
+        before(async () => {
+          await maybeClearSearchInput();
+          await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+          await ml.trainedModelsFlyout.assertFlyoutOpen();
+        });
+
+        after(async () => {
+          await maybeClearSearchInput();
+          await ml.testExecution.logTestStep('Close the Add Trained Model flyout');
+          await ml.trainedModelsFlyout.assertClosed();
+        });
+
+        it('should contain a Click to Download and a Manual Download tab', async () => {
+          await ml.testExecution.logTestStep(
+            'Assert the "Click to Download" and "Manual Download" tabs exists'
+          );
+          await ml.trainedModelsFlyout.assertTabsDifferPerUser('power');
+        });
+
+        it('should list Elser panels contents correctly', async () => {
+          await ml.testExecution.logTestStep('should display the Elser header copy');
+          await ml.trainedModelsFlyout.assertElserModelHeaderCopy();
+
+          await ml.testExecution.logTestStep('should display the Elser Panels');
+          await ml.trainedModelsFlyout.assertElserPanelsExist();
+
+          await ml.testExecution.logTestStep('should display the E5 Panels');
+          await ml.trainedModelsFlyout.assertE5PanelsExist();
+
+          await ml.testExecution.logTestStep('should display a Download Button');
+          await ml.trainedModelsFlyout.assertDownloadButtonExists();
+        });
+      });
+
+      it('should not be able to delete a model assigned to all spaces, and show a warning copy explaining the situation', async () => {
+        await ml.testExecution.logTestStep('should select the model named elser_model_2');
+        await testSubjects.click('checkboxSelectRow-.elser_model_2');
+
+        await ml.testExecution.logTestStep('should attempt to delete the model');
+        await find.clickByButtonText('Delete');
+
+        await ml.testExecution.logTestStep(
+          'assert the action is banned, specifically checking the error message'
+        );
+        await ml.trainedModelsTable.assertSpaceAwareWarningCopy();
+
+        await ml.testExecution.logTestStep('close the eui modal');
+        await find.clickByButtonText('Close');
```

Comment:
can be part of the trained models service

Reply:
Done